core/src/serve_repair.rs

```diff
@@ -161,6 +170,7 @@ pub enum RepairProtocol {
     WindowIndexWithNonce(ContactInfo, Slot, u64, Nonce),
     HighestWindowIndexWithNonce(ContactInfo, Slot, u64, Nonce),
     OrphanWithNonce(ContactInfo, Slot, Nonce),
+    CodingWithNonce(ContactInfo, Slot, u64, Nonce),
```

Comment:
This is not backward compatible. Please only add new enum values at the end.
If a node receives a serialized enum with tag larger than the number of enum cases it knows of, it simply fails deserialization which is fine.
But if you change the type of one of the enum cases, it will possibly succeed deserialization but only with a corrupted value.